core/native/src/main/scala/cats/effect/unsafe/IORuntimeConfigCompanionPlatform.scala

```diff
@@ -50,12 +50,32 @@ private[unsafe] abstract class IORuntimeConfigCompanionPlatform { this: IORuntim
         .flatMap(x => Try(x.toBoolean).toOption)
         .getOrElse(DefaultReportUnhandledFiberErrors)
 
+    val cpuStarvationCheckInterval =
+      Option(System.getenv("CATS_EFFECT_CPU_STARVATION_CHECK_INTERNAL"))
+        .map(Duration(_))
+        .flatMap { d => Try(d.asInstanceOf[FiniteDuration]).toOption }
+        .getOrElse(DefaultCpuStarvationCheckInterval)
+
+    val cpuStarvationCheckInitialDelay =
+      Option(System.getenv("CATS_EFFECT_CPU_STARVATION_CHECK_INITIALDELAY"))
```

Comment:
```suggestion
      Option(System.getenv("CATS_EFFECT_CPU_STARVATION_CHECK_INITIAL_DELAY"))
```

This makes me feel like we should find a way to share some of this between JS and Scala Native. Probably not for this PR but something to think about for a follow-up.

Reply:
Yes, would be nice!